packages/api-explorer/src/Doc.jsx

```diff
@@ -10,7 +10,6 @@ const Oas = require('oas/tooling');
 const { getPath, matchesMimeType } = require('oas/tooling/utils');
 
 const { TutorialTile } = require('@readme/ui/.bundles/es/ui/compositions');
-require('@readme/ui/.bundles/umd/main.css');
```

Comment:
This is definitely in keeping with the other CSS optimizations we've been making! But will be difficult to test until we run an integration, and I know we've had [some](https://github.com/readmeio/readme/blob/0fb6ae74028f78bfec5fec96208cb11d5b2a4879/packages/hub/public/css/bundle-hub2.styl#L74-L86) issue with cross-repo style application before.